independent-projects/resteasy-reactive/server/runtime/src/main/java/org/jboss/resteasy/reactive/server/handlers/ResponseWriterHandler.java

```diff
@@ -23,6 +23,7 @@ public ResponseWriterHandler(DynamicEntityWriter dynamicEntityWriter) {
 
     @Override
     public void handle(ResteasyReactiveRequestContext requestContext) throws Exception {
+        requestContext.requireCDIRequestScope();
```

Comment:
The commit that contains this change should be first in the PR, because if we take this away, the build would be failing

Reply:
I see it already in the first place. This is the first commit of this PR :/